scripts/common/commit-msg-regex.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Exposes the regex used to validate the Git commit message so it can be shared between scripts.
```

Comment:
If I'm not mistaken, the original file contains some example of compliant and non-compliant messages.
What do you think of moving them here?

Reply:
Yes, good suggestion, I'll move it here!

Reply:
Done!